frontend/src/app/modules/apiv3/endpoints/users/apiv3-users-paths.ts

```diff
@@ -41,4 +42,18 @@ export class Apiv3UsersPaths extends APIv3ResourceCollection<UserResource, APIv3
 
   // /api/v3/users/me
   public readonly me = this.path + '/me';
+
+  /**
+   * Create a new UserResource
+   *
+   * @param resource
+   */
+  public post(resource:UserResource|{firstName:string, email:string, status:'invited'}):Observable<UserResource> {
```

Comment:
Same comment as above. We can freely choose whether we want to pass a HAL resource here or the payload object itself. I'd vote for the payload object to make the interface clear